index.src.html

```diff
@@ -749,6 +755,166 @@ <h3 id="try-delivery" algorithm>
   8.  Return "`Failure`".
 </section>
 
+<section>
+  <h2 id="observers">Reporting Observers</h2>
+
+  A <dfn>reporting observer</dfn> observes some types of <a>reports</a> from
+  JavaScript, and is represented in JavaScript by the {{ReportingObserver}}
+  object.
+
+  Each <a>environment settings object</a> has a <dfn>registered reporting
+  observer list</dfn>, which is an <a>ordered set</a> of <a>reporting observers</a>.
+
+  Any <a>reporting observer</a> that is in a <a>registered reporting observer
+  list</a> is considered <dfn>registered</dfn>.
+
+  Each <a>environment settings object</a> has a <dfn>report buffer</dfn>, which
+  is an ordered list of <a>reports</a> that have been generated in
+  that <a>environment settings object</a>. This list is initially empty, and the
+  reports are stored in the same order in which they are generated.
```

Comment:
For PO we queue the pending entries / reports into each observer, instead of keeping a global buffer:
https://w3c.github.io/performance-timeline/#queue-a-performanceentry

Reply:
This buffer is just used for the purpose of passing past reports into new observers. Pending reports are still queued on observers (in their "report list").

Reply:
Is this to handle a race condition, where an observer might be registered after some of the events that it needs to monitor?  If so I suggest some explanatory text somewhere; the name `buffered` implied to me something about queueing up reports between when Reporting receives from the downstream spec and before they're handed over to the observer.